salt/modules/virt.py

```diff
@@ -2095,6 +2258,33 @@ def get_graphics(vm_, **kwargs):
     return graphics
 
 
+def get_loader(vm_, **kwargs):
+    '''
+    Returns the information on the loader for a given vm
+
+    :param vm_: name of the domain
+    :param connection: libvirt connection URI, overriding defaults
+
+        .. versionadded:: Fluorine
```

Comment:
You don't need to have a ``.. versionadded:: Fluorine`` on all these properties since the whole function is brand new. Just add one ``.. versionadded`` on the function level.